projects/go-lib/src/lib/components/go-file-upload/go-file-upload.component.html

```diff
@@ -0,0 +1,53 @@
+<label class="go-form__label"
+       [attr.for]="id"
+       [ngClass]="{'go-form__label--dark': theme === 'dark'}">
+  {{ label }}
+</label>
+<div
+  goDragonDrop
+  class="go-file-upload go-file-upload__container"
+  [ngClass]="{ 'go-file-upload--disabled': state === 'selected' || isLoading }"
+  (click)="filePicker.click()"
+  (fileDropped)="onFilePicked($event)"
+>
+  <div class="go-file-upload__container" *ngIf="state === 'selecting' && !isLoading">
+    <go-icon icon="publish" iconModifier="large"></go-icon>
+    Drag and Drop File or Click to Browse
+  </div>
+  <div class="go-file-upload__container go-file-upload__container--loading" *ngIf="isLoading">
+    <go-loader *ngIf="isLoading"></go-loader>
+    <span class="go-file-upload__container--loading-text">Uploading File...</span>
+  </div>
+  <div class="go-file-upload__container go-file-upload__container--selected" *ngIf="state === 'selected' && !isLoading">
+    <go-icon icon="check" iconModifier="positive" iconClass="go-icon--large"></go-icon>
```

Comment:
We might want to check if @jaredami's PR is going to affect the `iconModifer` here. #281

Reply:
I don't think my PR effects this. However, if I'm not mistaken, while the `go-button` takes "positive" as an option for its `buttonVariant` input (and converts it internally to "primary"), the `go-icon` only accepts "primary" for its `iconModifier` input, since it has no internal logic which converts "positive" to "primary" and only has a `go-icon--primary` class, and not a `go-icon--positive` class.

Reply:
@jaredami you make a good point here. I think it might make more sense to set the color via a class from the `go-file-upload` component itself. Thought's @StevenUlmer ?

Reply:
Ya that makes sense. I added a class in the `go-file-upload` component to set the color